apps/backoffice-v2/src/Router/Router.tsx

```diff
@@ -46,12 +48,12 @@ const router = createBrowserRouter([
             children: [
               ...(env.VITE_AUTH_ENABLED
                 ? [
-                    {
-                      path: '/:locale/auth/sign-in',
-                      element: <SignIn />,
-                      errorElement: <RouteError />,
-                    },
-                  ]
+                  {
+                    path: '/:locale/auth/sign-in',
+                    element: <SignIn />,
+                    errorElement: <RouteError />,
+                  },
+                ]
```

Comment:
Please ensure your IDE is configured to run Prettier and ESLint on save. Install the respective plugins if you haven't and make sure you're not running a custom format on save. 🙏🏼